io.github.brunoherbelin.Vimix.json

```diff
@@ -0,0 +1,159 @@
+                "-DCMAKE_BUILD_TYPE=Release",
+                "-DBUILD_SHARED_LIBS=ON",
+                "-DGLFW_BUILD_EXAMPLES=OFF",
+                "-DGLFW_BUILD_TESTS=OFF",
+                "-DGLFW_BUILD_DOCS=OFF"
+            ],
+            "sources": [
+                {
+                    "type": "archive",
+                    "url": "https://github.com/glfw/glfw/releases/download/3.3.8/glfw-3.3.8.zip",
+                    "sha256": "4d025083cc4a3dd1f91ab9b9ba4f5807193823e565a5bcf4be202669d9911ea6"
+                }
+            ]
+        },
+        {
+            "name": "x264",
+            "config-opts": [
+                "--enable-shared"
+            ],
+            "sources": [
+            {
+                "type": "git",
+                "commit": "baee400fa9ced6f5481a728138fed6e867b0ff7f",
+                "url": "https://code.videolan.org/videolan/x264.git"
+            }
+            ]
+        },
+        {
+            "name": "srt",
+            "buildsystem": "cmake",
+            "config-opts": [
+                "-DCMAKE_BUILD_TYPE=Release",
+                "-DENABLE_APPS=OFF",
+                "-DENABLE_STDCXX_SYNC=ON"
+            ],
+            "sources": [
+            {
+                "type": "git",
+                "tag": "v1.5.1",
+                "commit": "0bc3b03202b3159fc9b085b3ae6d66ec071c25d6",
+                "url": "https://github.com/Haivision/srt.git"
+            }
+            ]
+        },
+        {
+            "name": "frei0r",
+            "buildsystem": "cmake",
+            "config-opts": [
+                "-DCMAKE_BUILD_TYPE=Release"
+            ],
+            "sources": [
+            {
+                "type": "git",
+                "tag": "v1.8.0",
+                "commit": "c11beecfaad5c29350127515795295a13d53e956",
+                "url": "https://github.com/dyne/frei0r.git"
+            }
+            ]
+        },
+        {
+            "name": "gstreamer",
+            "buildsystem": "meson",
+            "builddir": true,
+            "config-opts": [
+               "-Ddoc=disabled",
+               "-Dpython=disabled",
+               "-Dges=disabled",
+               "-Dgst-examples=disabled",
+               "-Drtsp_server=disabled",
+               "-Ddevtools=disabled",
+               "-Dgpl=enabled",
+               "-Dlibav=enabled",
+               "-Dbad=enabled",
+               "-Dgst-plugins-bad:srt=enabled",
+               "-Dugly=enabled",
+               "-Dgst-plugins-ugly:x264=enabled"
+            ],
+            "sources": [
+              {
+                "type": "git",
+                "tag": "1.20.4",
+                "commit": "642730bd67d3c23ac68bd328fbddd191a7a43f25",
+                "url": "https://gitlab.freedesktop.org/gstreamer/gstreamer.git",
+                "disable-submodules": false
+              }
+            ]
+        },
+        {
+            "name": "shmdata",
+            "buildsystem": "cmake",
```

Comment:
```suggestion
            "buildsystem": "cmake-ninja",
```

Reply:
and this/

Reply:
vimix allows exporting graphics in shared memory with the library shmdata
This specific library is used for compatibility with the software Splash for video mapping projection.

Reply:
Tested and compiling with cmake-ninja